usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -154,16 +160,20 @@ def create_city_name_queryset(scope: str, list_of_cities: list, country_code: st
     state_code (optional) is the state code if the search should be limited to that state
     """
     matching_awards = set(
-        chain(*[get_award_ids_by_city(scope, city, country_code, state_code) for city in list_of_cities])
+        chain(*[get_award_ids_by_city(
+            scope, desired_id_field, city, country_code, state_code) for city in list_of_cities]
+        )
     )
     result_queryset = Q(pk=None)  # If there are no city results in Elasticsearch, use this always falsey Q filter
 
     if matching_awards:
-        result_queryset = Q(**{"award_id" + "__in": matching_awards})
+        result_queryset = Q(**{desired_id_field + "__in": matching_awards})
     return result_queryset
 
 
-def get_award_ids_by_city(scope: str, city: str, country_code: str, state_code: Optional[str] = None) -> list:
+def get_award_ids_by_city(
```

Comment:
This method name might be a little confusing being that sometimes it returns transaction ids and not award ids.